functions-and-operators/tidb-functions.md

```diff
@@ -22,7 +22,9 @@ TableStmt ::=
     "TIDB_DECODE_KEY(" STR ")"
 ```
 
-### Example
+### Examples
+
+In the first example we have a table `t1` which has a hidden rowid that is generated by TiDB. This is what happens for a non-clustered primary key.
```

Comment:
```suggestion
In the following example, the table `t1` has a hidden `rowid` that is generated by TiDB. The `TIDB_DECODE_KEYSQL` is used in the statement. From the result, you can see that the hidden `rowid` is decoded and output, which is a typical result for the non-clustered primary key.
```

Reply:
I changed `TIDB_DECODE_KEYSQL` to `TIDB_DECODE_KEY` but otherwise applied the change.